assets/sass/05-blocks/group/_editor.scss

```
@@ -25,7 +25,7 @@
 		padding: var(--global--spacing-vertical);
 	}
 
-	.wp-block-group__inner-container *:last-child {
+	.wp-block-group__inner-container > :last-child {
```

```suggestion
	.wp-block-group__inner-container > *:last-child {
```

This is probably personal style, but I find these pseudo-classes easier to read with the `*`, and it doesn't affect the specificity (since `*` has none).